server/Arcadia.Assistant.CSP/CspEmployeesInfoStorage.cs

```diff
@@ -32,7 +32,7 @@ protected override async Task<LoadAllEmployees.Response> GetAllEmployees()
                             HireDate = x.HiringDate,
                             FireDate = x.FiringDate,
                             MobilePhone = x.MobilePhone,
-                            RoomNumber = x.RoomNumber,
+                            RoomNumber = x.RoomNumber != null ? x.RoomNumber.Trim() : null,
```

Comment:
I wonder how that happened... apparently someone needs to merge things back from master to develop :)

Reply:
Done. master -> develop -> current branch.